docs/modeling/bounding-boxes.rst

```diff
@@ -0,0 +1,216 @@
+.. _bounding-boxes:
+
+Efficient Model Rendering with Bounding Boxes 
+=============================================
+
+.. versionadded:: 1.1
+
+All `astropy.modeling.Model` subclasses have a ``bounding_box`` attribute that 
+can be used to set the limits over which the model is significant. This greatly 
+improves the effciency of evaluation when the input range is much larger than 
+the characteristic width of the model itself. For example, to create a sky model 
+image from a large survey catalog, each source should only be evaluated over the 
+pixels to which it contributes a significant amount of flux. This task can 
+otherwise be computationally prohibitive on an average CPU. 
+
+The `astropy.modeling.render_model` function can be used to evaluate a model on 
+an input array, or coordinates, limiting the evaluation to the ``bounding_box`` 
+region if it is set.
+
+Using the Bounding Box
+-----------------------
+
+``bounding_box`` is a property that returns a tuple of tuples defining the 
+limits for each input. ``((y_low, y_high), (x_low, x_high), (z_low, z_high))``. 
+Note that the ``'y'``-coordinate 
+comes first for ``model.n_inputs > 1`` following the `numpy` array indexing 
+convention.
```

Comment:
I know that the numpy axis convention implies `(y, x)`.
It's not clear to me why it's `(y, x, z)` instead of  `(z, y, x)` or where a fourth dimension would go.
Maybe it's best to only mention the common 1D and 2D case here and then have a separate paragraph about "bounding box tuple axis order" later? (alternatively, add one or two sentences explaining it some more here)